src/filters/dates/partialDates.test.ts

```diff
@@ -0,0 +1,143 @@
+import { DateTime } from 'luxon';
+import { TestCaseSchema } from '../../serina.schema';
+import PartialDates from './partialDates';
+
+const currentYear = DateTime.utc().year;
+
+const mockDates = (day, month, year) => DateTime.utc()
+    .set({ day, month, year })
+    .endOf('day')
+    .toJSDate();
+
+describe('Dates', () => {
+    describe('Partial Dates', () => {
+        const partialDateTestCases: TestCaseSchema[] = [{
+            case: 'go to work on 02/2009',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on 02/2009' },
+            ],
+        }, {
+            case: 'go to work on Feb 2009',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on Feb 2009' },
+            ],
+        }, {
+            case: 'go to work on February 2009',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on February 2009' },
+            ],
+        }, {
+            case: 'go to work on 2/2009',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on 2/2009' },
+            ],
+        }, {
+            case: 'go to work on 2009 Feb',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on 2009 Feb' },
+            ],
+        }, {
+            case: 'go to work on 2009 February',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on 2009 February' },
+            ],
+        }, {
+            case: 'go to work on 2009/2',
+            result: [
+                { dateTime: mockDates(1, 2, 2009), text: 'go to work', matched: 'on 2009/2' },
+            ],
+        }, {
+            case: 'go to work on 02/20',
+            result: [
+                { dateTime: mockDates(20, 2, currentYear + 1), text: 'go to work', matched: 'on 02/20' },
```

Comment:
`currentYear + 1` is not going to work once we get to 2020, we need to mock the date at the top

Reply:
See https://github.com/h-dong/Serina/blob/master/src/filters/day/day.test.ts#L14

Reply:
I did it in the other test file, just forgot to put it here as well.